plugins/moz_emr/emr_create_job_flow_selective_template_operator.py

```diff
@@ -0,0 +1,25 @@
+from airflow.contrib.operators.emr_create_job_flow_operator import EmrCreateJobFlowOperator
+from airflow.utils import apply_defaults
+
+from moz_emr.moz_emr_mixin import MozEmrMixin
+
+
+class EmrCreateJobFlowSelectiveTemplateOperator(MozEmrMixin, EmrCreateJobFlowOperator):
```

Comment:
If there were less quirkiness with the templating engine, would this class still be necessary for mixing the mixins?

Reply:
At the moment, yeah -- unless I can figure out a way to get the utility functions in the path through official channels, this will probably have to stay